tests/test_convert.py

```diff
@@ -70,9 +71,12 @@ def test_azure_paragraph_to_text_block(document_paragraph: DocumentParagraph) ->
     assert text_block.text_block_id == "1"
     assert document_paragraph.bounding_regions is not None
     assert text_block.page_number == document_paragraph.bounding_regions[0].page_number
-    assert text_block.coords == polygon_to_co_ordinates(
-        document_paragraph.bounding_regions[0].polygon
-    )
+    assert text_block.coords == [
```

Comment:
Is there a better way of doing this that isn't just effectively re-writing a function? 

Just having a set expected output like [(),(),()]?

Reply:
not that I can think of